.github/workflows/python-test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip setuptools wheel
-          pip install -r requirements.txt
+          pip install -r test-requirements.txt
```

Comment:
That's right. This worked previously because `test-requirements.txt` contained the modules defined in `requirements.txt`. Now, will need to have both.